jsbasics/palindrome.js

```diff
@@ -0,0 +1,12 @@
+let readline = require("readline-sync");
+let str = readline.question("enter word: ");    // use to enter user input word
+
+const palindrome = str => {
+  str = str.toLowerCase().replace(/[^a-z]+/g, "");    // tolowercase - use to convert into lowerstring., replace() - return matches of a pattern replaced by a replacement
+  if (str === str.split("").reverse().join("")) {     //  ,"" remove spaces, /[^a-z] - convert the string into lowercase, 
```

Comment:
you should do this without using string functions

Reply:
In today's call, you mentioned if you're ok with string functions you can use string functions. Could you please assist me is this necessary to change this program.